doc/index.php

```diff
@@ -12,11 +12,11 @@
 
 <p><ul>
 
-<li><a href="current/"><strong>v4.1 series</strong></a>.
-This documentation reflects the latest progression in the 4.1.x series.
+<li><a href="current/"><strong>v5.0 series</strong></a>.
```

Comment:
Shouldn't this go out to docs.open-mpi.org?

Reply:
you're right, fixed with forced update